budget_management_project/expense/serializers.py

```diff
@@ -0,0 +1,37 @@
+from django.utils import timezone
+from rest_framework import serializers
+
+from expense.models import Expense
+
+class ExpenseSerializer(serializers.ModelSerializer):
+    category_name = serializers.CharField(source='category.name', read_only=True)
```

Comment:
이 부분 read_only를 Meta 클래스에 read_only_field를 한번에 명시해주시면 좋을 것 같습니다.
```suggestion
    category_name = serializers.CharField(source='category.name')
```